collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
@tabladrum what do you think of the idea of having a pseudo scopeowner (Collector item) representing the "any" option? This way we could still have an attached collector item even if "Any" is selected (if there are teams and any is selected or if there are no teams and it is the only option). It works because the feature documents are not attached to collector items and instead we query them by fields.